plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/editor/KotlinLiteralCopyPasteProcessor.kt

```diff
@@ -149,22 +149,25 @@ class KotlinLiteralCopyPasteProcessor : CopyPastePreProcessor {
                     beginTp.getQualifiedExpressionForReceiver()?.callExpression?.calleeExpression?.text == "trimIndent" &&
                     templateTokenSequence.firstOrNull()?.indent() == templateTokenSequence.lastOrNull()?.indent()
                 ) {
-                    begin.parent?.prevSibling?.takeIf { it.text != "\n" && it.text != "\"\"\""  }?.text
+                    begin.parent?.prevSibling?.text?.takeIf { it.all { c -> c == ' ' } }
```

Comment:
Some people use tabs instead of spaces for indentation

Reply:
Also this line of code isn't covered by tests but it changes the behaviour - https://youtrack.jetbrains.com/issue/KTIJ-15936